OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -202,6 +325,7 @@ Properties trackVideoOrientation(String videoId, Double currentTime,
         String VIDEO_DOWNLOADED = "edx.bi.video.downloaded";
         String USERLOGOUT = "edx.bi.app.user.logout";
         String USERLOGIN = "edx.bi.app.user.login";
+        String USER_REGISTER = "edx.bi.app.user.register";
```

Comment:
This is a new event that the OSPR had:

`public void trackUserRegister(String method, boolean didSucceed)`

For backward compatibility, we should probably stick to our previous event i.e. 

`public void trackCreateAccountClicked(String appVersion, String source)`